source/dev-setup.rst

```diff
@@ -55,6 +55,8 @@ PostgreSQL
 
 The first important package to install from Homebrew is the PostgreSQL database and the PostGIS
 extensions::
+*note that "brew install postgis" will install the latest version of postgis along with the latest
+version of potstgres
```

Comment:
Can you add something about the ramifications of this and what someone can do about it?

Also, this text has been added between the "::" at the end of a paragraph that starts a code example, and the code example; the formatting will need to be adjusted one way or another.